index.html

```diff
@@ -13,8 +14,24 @@
         </div>
             
         <div class="menu">
-            Menu
+            <div>
+            <i class="fas fa-bars icon"></i>
+            </div>
```

Comment:
Making every html tag in one indent can cause a lot of frustration in the future. For the moment it is a good practice to train your hand to write clean code without extensions.

```suggestion
            <div>
            	<i class="fas fa-bars icon"></i>
            </div>
```